contracts/axelarnet-gateway/src/contract/execute.rs

```diff
@@ -13,7 +13,7 @@ use crate::executable::AxelarExecutableClient;
 use crate::state::{self};
 
 #[allow(clippy::too_many_arguments)]
-pub(crate) fn call_contract(
+pub fn call_contract(
```

Comment:
It's more common to manage visibility at a level higher since the module doesn't need to worry about whether the method is used only within the crate or externally.